build/templates/workbook.tpl.json

```diff
@@ -126,6 +126,7 @@
             "name": "Help",
             "label": "Show Help",
             "type": 10,
+            "description": "This will show more information to help you understand resiliency the focus area",
```

Comment:
This will show the additional reliability guidance for the focus area to help you decide on the right reliability posture of the service